### DISCUSSION TITLE
How to add non auth custom admin page?

### DISCUSSION THREAD

**Discussion by ilyaskarim** [keystonejs/keystone#7788] [Questions]
I am assigned a task to create a non-auth custom page to allow users to accept the invite when the user is created into CMS, is this their way to do it? as far as I am aware we cannot create pages for non-auth users.

**Comment by flexdinesh** [comment]
You absolutely can. You gotta create a public page and make sure access control is skipped for that page and build your requirements in that page.

Example codesandbox here — https://codesandbox.io/s/access-control-and-public-pages-uk06ez
